.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# These owners will be the default owners for everything in
+
+# the repo. Unless a later match takes precedence,
+
+# @salesforcecli/cli will be requested for
+
+# review when someone opens a pull request.
+
```

Comment:
IS this formatted correctly?  It looks different then https://github.com/forcedotcom/salesforcedx-templates/blob/main/.github/CODEOWNERS

Reply:
I think it would probably still work. Github is showing it as valid, but lets updated it to match [what is in our plugin template](https://github.com/salesforcecli/plugin-template-sf/blob/main/.github/CODEOWNERS)